packages/fxa-payments-server/src/components/LoadingSpinner.tsx

```diff
@@ -6,7 +6,7 @@ const spinnerImage =
 
 export const LoadingSpinner = () => (
   <div id="loading-spinner" data-testid="loading-spinner">
-    <img alt="Loading" src={spinnerImage} />
```

Comment:
This is kinda controversial i guess, but I don't think this image sticks around long enough for the alt text to have semantic importance (it could easily be a CSS background), and sometimes the word "Loading" flashes briefly on the screen on refresh

Reply:
For this to be technically valid HTML, we should at least set it to an empty string (`alt=''`).

Reply:
Kk, I can fix that